src/libsyntax/feature_gate.rs

```diff
@@ -378,6 +374,8 @@ declare_features! (
     // Allows `..` in tuple (struct) patterns
     (accepted, dotdot_in_tuple_patterns, "1.14.0", Some(33627)),
     (accepted, item_like_imports, "1.14.0", Some(35120)),
+    // Allows using `Self` and associated types in struct expressions and patterns.
+    (accepted, more_struct_aliases, "1.14.0", Some(37544)),
```

Comment:
I see, let's see if anyone can say if there's a reason to update it. For library features it's neat to see it in the docs, I guess here it doesn't actually show anywhere?